src/box/lua/schema.lua

```diff
@@ -2586,6 +2591,10 @@ space_mt.upgrade = function(space, ...)
     check_space_arg(space, 'upgrade')
     return box.schema.space.upgrade(space.id, ...)
 end
+space_mt.wal_ext = function(space, ...)
+    check_space_arg(space, 'wal_ext')
+    return box.schema.space.wal_ext(space.id, ...)
+end
```

Comment:
Why do you need this?

Reply:
Removed. It was simple and useful handle to introspect space wal format.